metabase/metabase
# fix(sdk): hide downloads by default

### Description

As we have issues on [visual artifacts](https://github.com/metabase/metabase/issues/48716) in PNG/PDF image downloads in the SDK when custom themes are being applied, we decided to hide all downloads by default.

### How to verify

- Render a StaticDashboard or InteractiveDashboard
- Both the PDF exports and the dashcard download should be disabled by default.
- Using the `withDownloads` prop should enable the downloads.
